CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing to ArmoniK
+
+Hi! We are excited that you are interested in ArmoniK and want to make it even better! This document is a general guide we would like our contributors to follow. Before submitting your contribution, please make sure to take a moment and read through the following guide.
+
+## Discuss First !
+
+Before attempting to make a feature pull request, open an issue and discuss it with us. It will help maintainers understand what you want to do as well as make sure that you go in a direction that is suitable for ArmoniK' future.
+
+## Submission guidelines
+
+### Submit an issue
+
+Before submitting a new issue, please take a look at the existing [ArmoniK issues](https://github.com/aneoconsulting/ArmoniK/issues) to see if someone already mentioned your issue. If it already exists, it may inform you of workaround readily available.
+
+We want to fix issues as soon as possible, but before fixing a bug, we need to reproduce and confirm it. In order to do so, we require you to provide a minimal reproduction example. This will give us important information without having to ask you multiple complementary questions.
+
+A minimal reproductoion scenario allows us to quickly confirm a bug, isolate it and confirm that we are really fixing it.
+
+Unfortunately, we are not able to investigate or fix bugs without a minimal reproduction sample, so if we don't hear back from you, we will close the issue since it is missing information to be reproduced.
```

Comment:
if user can provide more informations in after the issue was closed, we can reopened it.